hydra-tui/test/Hydra/TUISpec.hs

```diff
@@ -38,15 +38,19 @@ import Hydra.Cluster.Fixture (
   aliceSk,
  )
 import Hydra.Cluster.Util (chainConfigFor, keysFor)
-import Hydra.ContestationPeriod (toNominalDiffTime)
+import Hydra.ContestationPeriod (ContestationPeriod (UnsafeContestationPeriod), toNominalDiffTime)
 import Hydra.Logging (showLogsOnFailure)
 import Hydra.Network (Host (..))
 import Hydra.Options (ChainConfig (..))
-import Hydra.TUI (renderTime, runWithVty, tuiContestationPeriod)
+import Hydra.TUI (renderTime, runWithVty)
 import Hydra.TUI.Options (Options (..))
 import HydraNode (EndToEndLog, HydraClient (HydraClient, hydraNodeId), withHydraNode)
 import System.Posix (OpenMode (WriteOnly), closeFd, defaultFileFlags, openFd)
 
+-- TODO(SN): hardcoded contestation period used by the tui
```

Comment:
I think this is not a TODO anymore -> no problem to use hard-coded values in tests.